conjure-java-core/src/integrationInput/java/com/palantir/product/ErrorServiceBlocking.java

```diff
@@ -0,0 +1,433 @@
+            public TestEmptyBodyResponse testEmptyBody(AuthHeader authHeader, boolean shouldThrowError) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams("Authorization", authHeader.toString());
+                _request.body(testEmptyBodySerializer.serialize(shouldThrowError));
+                return _runtime.clients()
+                        .callBlocking(testEmptyBodyChannel, _request.build(), testEmptyBodyDeserializer);
+            }
+
+            @Override
+            public TestBinaryResponse testBinary(AuthHeader authHeader, boolean shouldThrowError) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams("Authorization", authHeader.toString());
+                _request.body(testBinarySerializer.serialize(shouldThrowError));
+                return _runtime.clients().callBlocking(testBinaryChannel, _request.build(), testBinaryDeserializer);
+            }
+
+            @Override
+            public TestOptionalBinaryResponse testOptionalBinary(
+                    AuthHeader authHeader, OptionalBinaryResponseMode mode) {
+                Request.Builder _request = Request.builder();
+                _request.putHeaderParams("Authorization", authHeader.toString());
+                _request.body(testOptionalBinarySerializer.serialize(mode));
+                return _runtime.clients()
+                        .callBlocking(testOptionalBinaryChannel, _request.build(), testOptionalBinaryDeserializer);
+            }
+
+            @Override
+            public String toString() {
+                return "ErrorServiceBlocking{_endpointChannelFactory=" + _endpointChannelFactory + ", runtime="
+                        + _runtime + '}';
+            }
+        };
+    }
+
+    /** Creates an asynchronous/non-blocking client for a ErrorService service. */
+    static ErrorServiceBlocking of(Channel _channel, ConjureRuntime _runtime) {
+        if (_channel instanceof EndpointChannelFactory) {
+            return of((EndpointChannelFactory) _channel, _runtime);
+        }
+        return of(
+                new EndpointChannelFactory() {
+                    @Override
+                    public EndpointChannel endpoint(Endpoint endpoint) {
+                        return _runtime.clients().bind(_channel, endpoint);
+                    }
+                },
+                _runtime);
+    }
+
+    final class Factory implements DialogueServiceFactory<ErrorServiceBlocking> {
+        @Override
+        public ErrorServiceBlocking create(EndpointChannelFactory endpointChannelFactory, ConjureRuntime runtime) {
+            return ErrorServiceBlocking.of(endpointChannelFactory, runtime);
+        }
+    }
+
+    sealed interface TestBasicErrorResponse
+            permits TestBasicErrorResponse.Success, TestBasicErrorResponse.InvalidArgument {
+        record Success(@JsonValue String value) implements TestBasicErrorResponse {
```

Comment:
Another change since last review: I added the @JsonValue. This is needed to deserialize the records (similar to `@JsonCreator(mode = Mode.DELEGATING)`): [this is a related issue.](https://github.com/FasterXML/jackson-databind/issues/3180#issuecomment-1306160265)